test/black-box/address/AddressAddCommand.test.ts

```diff
@@ -103,10 +104,9 @@ describe('AddressAddCommand', () => {
             testData.zipCode
         ]);
         res.expectJson();
-        res.expectStatusCode(400);
+        res.expectStatusCode(404);
         expect(res.error.error.success).toBe(false);
```

Comment:
these expects: expect(res.error.error.success).toBe(false); are unnecessary and slow down development as you don't ever get to the next expect which would tell you what exactly went wrong. res.error.error.success is always false when res.error.error.message exists.